recipes/y-py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "y-py" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yjs/y-crdt/archive/v{{ version }}.tar.gz
+  sha256: df7ad800517766c2b73b02e3c07679c04d107f2b12c59dd7d0f6c3513a3918c6
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install ./y-py -vv
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+requirements:
+  build:
+    - {{ compiler('gcc') }}  # [linux]
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - maturin
```

Comment:
it may be worth having a bottom pin on `maturin`, if it is known... i was bitten by this someplace...